libs/ui-components/package.json

```diff
@@ -27,7 +27,7 @@
     "yup": "^1.3.3"
   },
   "peerDependencies": {
-    "@patternfly/react-core": "^5.0.0",
+    "@patternfly/react-core": "^5.3.0",
```

Comment:
we should not go higher than OCP react-core https://github.com/openshift/console/blob/master/frontend/package.json#L156 otherwise in our ACM plugin, new features from react-core wont be available